recipe/meta.yaml

```
@@ -20,11 +19,9 @@ build:
     - '*libnvidia-ml.so*'
     - '*libnccl.so*'
   ignore_run_exports_from:
-    # CXX standard library is unused
-    - {{ compiler('cxx') }}
     # allow CPU-only use cases (xref: conda-forge/ucx-split-feedstock#173)
     - {{ compiler("cuda") }}
-    - cuda-cudart-dev  # [(cuda_compiler_version or "None").startswith("12")]
+    - cuda-cudart-dev  # [cuda_compiler == "cuda-nvcc"]
```

As this selector was only needed for CUDA 11 and we have already dropped CUDA 11 support, we can just drop this selector

```suggestion
    - cuda-cudart-dev
```